sql/sqlite/driver.go

```diff
@@ -60,6 +62,15 @@ func init() {
 			return uc
 		})),
 	)
+	sqlclient.Register(
+		LibsqlDriverName,
+		sqlclient.DriverOpener(Open),
+		sqlclient.RegisterCodec(MarshalHCL, EvalHCL),
+		sqlclient.RegisterURLParser(sqlclient.URLParserFunc(func(u *url.URL) *sqlclient.URL {
+			uc := &sqlclient.URL{URL: u, DSN: "wss://" + strings.TrimPrefix(u.String(), u.Scheme+"://"), Schema: mainFile}
+			return uc
+		})),
+	)
```

Comment:
I think a better way, that will also support our platform is to register a flavor (of `sqlite`) named `libsql` and just handle it differently using the URL parser/opener. Otherwise, this will require us to handle libsql in all our other integrations - which is much bigger effort.